hack/build/build-manifests.sh

```diff
@@ -18,93 +18,35 @@ script_dir="$(readlink -f $(dirname $0))"
 source "${script_dir}"/common.sh
 source "${script_dir}"/config.sh
 
-templates="$(find "${MANIFEST_TEMPLATE_DIR}" -name "*.in" -type f)"
 generator="${BIN_DIR}/manifest-generator"
 
 (cd "${CDI_DIR}/tools/manifest-generator/" && go build -o "${generator}" ./...)
 
-mkdir -p "${MANIFEST_GENERATED_DIR}/"
 
-for tmpl in ${templates}; do
-    tmpl=$(readlink -f "${tmpl}")
-    outFile=$(basename -s .in "${tmpl}")
-    rm -rf "${MANIFEST_GENERATED_DIR}/${outFile}"
-    (${generator} -template="${tmpl}" \
-        -docker-repo="${DOCKER_REPO}" \
-        -docker-tag="${DOCKER_TAG}" \
-        -controller-image="${CONTROLLER_IMAGE_NAME}" \
-        -importer-image="${IMPORTER_IMAGE_NAME}" \
-        -cloner-image="${CLONER_IMAGE_NAME}" \
-        -apiserver-image=${APISERVER_IMAGE_NAME} \
-        -uploadproxy-image=${UPLOADPROXY_IMAGE_NAME} \
-        -uploadserver-image=${UPLOADSERVER_IMAGE_NAME} \
-        -operator-image=${OPERATOR_IMAGE_NAME} \
-        -verbosity="${VERBOSITY}" \
-        -pull-policy="${PULL_POLICY}" \
-        -namespace="${NAMESPACE}"
-    ) 1>"${MANIFEST_GENERATED_DIR}/${outFile}"
+source "${script_dir}"/resource-generator.sh
+
 
-    (${generator} -template="${tmpl}" \
-        -docker-repo="{{ docker_prefix }}" \
-        -docker-tag="{{ docker_tag }}" \
-        -controller-image="{{ controller_image }}" \
-        -importer-image="{{ importer_image }}" \
-        -cloner-image="{{ cloner_image }}" \
-        -apiserver-image="{{ apiserver_image }}" \
-        -uploadproxy-image="{{ uploadproxy_image }}" \
-        -uploadserver-image="{{ uploadserver_image }}" \
-        -operator-image="{{ operator_image }}" \
-        -verbosity="{{ verbosity }}" \
-        -pull-policy="{{ pull_policy }}" \
-        -namespace="{{ cdi_namespace }}"
-    ) 1>"${MANIFEST_GENERATED_DIR}/${outFile}.j2"
-done
+mkdir -p "${MANIFEST_GENERATED_DIR}/"
```

Comment:
I thought we were going to move this stuf under `_out` as wll?

From `common.sh`: `MANIFEST_GENERATED_DIR=${CDI_DIR}/manifests/generated`

Reply:
Looks like KubeVirt puts similar stuff in `manifests/generated` so I understand if you want to keep as-is for now